app/_kong_plugins/ai-prompt-guard/index.md

```diff
@@ -42,4 +42,28 @@ search_aliases:
   - model
```

Comment:
```yaml
related_resources:
  - text: AI Semantic Prompt Guard plugin
    url: /plugins/ai-semantic-prompt-guard/
```
I'd add this as a related link since the two plugins are closely related in terms of blocking content.